api/Trace/Span.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Trace;
+
+interface Span extends SpanStatus, SpanKind
+{
+    function getSpanName(): string;
+    function getContext(): SpanContext;
+    function getParent(): ?SpanContext;
+    function getStartTimestamp(): string;
```

Comment:
Do you think we should use DateTime objects instead of strings?

Reply:
No. At least in my opinion this is a case where the cost of an object is too high:

```php
<?php
function cost_date() {
    $before = memory_get_usage();
    $date = new DateTime('now');
    $after = memory_get_usage();
    return [$date, $after - $before];
}

function cost_string() {
    $before = memory_get_usage();
    $date = microtime();
    $after = memory_get_usage();
    return [$date, $after - $before];
}

echo cost_date()[1], "\n";
echo cost_string()[1], "\n";
```
> 960
48

The object is just so heavy, and keep in mind you need a start and stop for every span.

Reply:
Although, "string" is pretty ambiguous, so I don't know. I'll think more on this.